addon/serializers/json-api.js

```diff
@@ -178,7 +179,11 @@ const JSONAPISerializer = JSONSerializer.extend({
   */
   pushPayload(store, payload) {
     let normalizedPayload = this._normalizeDocumentHelper(payload);
-    store.push(normalizedPayload);
+    if (isEnabled('ds-pushpayload-return')) {
```

Comment:
Can we do this in the store instead, and lets have the serializer return normalized json payload